modules/ROOT/pages/index.adoc

```diff
@@ -1,27 +1,13 @@
-= Bonita Intelligent Continuous Improvement Add-on
-:page-aliases: release_notes.adoc
+= Labs Program
```

Comment:
ℹ️ This is a reminder
We should try to add a `:description:` attribute in all pages to improve the SEO. See https://github.com/bonitasoft/bonitasoft.github.io/issues/129 for rationale.
The BICI documentation conversion has been managed prior we automate this metadata creation during the conversion process, so it is missing here.